### DISCUSSION TITLE
blog/parse-currency-amount-from-string-javascript

### DISCUSSION THREAD

**Discussion by giscus** [AshikNesin/nesinio-comments#20] [Announcements]
# blog/parse-currency-amount-from-string-javascript

Learn how to parse Currency from String in Javascript

https://nesin.io/blog/parse-currency-amount-from-string-javascript

**Comment by dm185016** [comment]
You should never ever use parseFloat for currency.  Too many rounding errors.  See https://currency.js.org/

**Reply by AshikNesin** [reply]
Yeah, I agree, @dm185016 

Using parseFloat might cause rounding errors with currency for some cases. I'll update the blog post to recommend better options. Thank you for your feedback.
